runner/src/simulate.rs

```diff
@@ -83,6 +87,31 @@ pub struct Opts {
     target_request_rate: Option<usize>,
 }
 
+#[derive(Debug, Clone, ValueEnum)]
+pub enum LogLevel {
+    Off,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
+
+impl LogLevel {
+    fn is_enabled(&self) -> bool {
+        !matches!(self, LogLevel::Off)
+    }
+
+    fn as_goose_log_level(&self) -> u8 {
+        match self {
+            LogLevel::Off => panic!("LogLevel::Off should not be used as an integer (it is None)."),
```

Comment:
uhm, huh? This is mimicking goose functionality?

Reply:
🙈 yeah, this was dumb. I thought I needed a value at first, so rather than use None I set it to off and then did this weird, "call this function to make sure it's not one variant and then do normal things. If you didn't follow this contract I will blow up in your face!" This was to deal with the fact that you can't set `GooseConfig.log_level = None`, but it turns out there's a much easier way.. which is use none lol. So now it does that.